
--- STDERR ---
Traceback (most recent call last):
  File "/home/sh1kaku/lihao/conkas/conkas.py", line 137, in <module>
    main()
  File "/home/sh1kaku/lihao/conkas/conkas.py", line 88, in main
    contracts = compile_files([filename])
  File "/home/sh1kaku/lihao/conkas/venv/lib/python3.10/site-packages/solcx/main.py", line 118, in compile_files
    stdoutdata, stderrdata, command, proc = solc_wrapper(**compiler_kwargs)
  File "/home/sh1kaku/lihao/conkas/venv/lib/python3.10/site-packages/solcx/utils/string.py", line 78, in inner
    return force_obj_to_text(fn(*args, **kwargs))
  File "/home/sh1kaku/lihao/conkas/venv/lib/python3.10/site-packages/solcx/wrapper.py", line 187, in solc_wrapper
    raise SolcError(
solcx.exceptions.SolcError: An error occurred during execution
> command: `/home/sh1kaku/.solcx/solc-v0.5.17 --combined-json abi,asm,ast,bin,bin-runtime,devdoc,opcodes,userdoc ../../SCRUBD/SCRUBD-CD/data/solidity_codes/0x8e3e25958ad448aeb5982a01b63bcb91355d56a5.sol`
> return code: `1`
> stderr:

> stdout:
../../SCRUBD/SCRUBD-CD/data/solidity_codes/0x8e3e25958ad448aeb5982a01b63bcb91355d56a5.sol:55:9: Error: "send" and "transfer" are only available for objects of type "address payable", not "address".
beneficiary.send(msg.value);
^--------------^
